networkx/algorithms/approximation/distance_measures.py

```diff
@@ -39,6 +39,17 @@ def diameter(G, seed=None):
     d : integer
        Lower Bound on the Diameter of G
 
+    Examples
+    --------
+    >>> G = nx.path_graph(10) #undirected graph
+    >>> nx.diameter(G)
+    9
+
+    >>> G = nx.DiGraph()
+    >>> G.add_edges_from([(2,3),(3,5),(5,2)]) #directed graph
```

Comment:
The cycle_graph function can be used for this case:
```suggestion
    >>> G = nx.cycle_graph([2, 3, 5], create_using=nx.DiGraph)  # directed graph
```